tests/io/readers/testPointListReader.cpp

```diff
@@ -91,9 +91,29 @@ bool testPointListReader()
     }
     trace.info()<< std::endl;
   }
-  nbok += (vectPolygons.at(0).size()==2 && vectPolygons.at(1).size()==2 && vectPolygons.at(0).at(0)==Z3i::Point(1,2,3)
-           && vectPolygons.at(0).at(1)==Z3i::Point(4,5,6) && vectPolygons.at(1).at(0)==Z3i::Point(0,0,10)
-           && vectPolygons.at(1).at(1)==Z3i::Point(0,5,5)) ? 1 : 0; 
+  trace.beginBlock ( "Testing reading spectial 1 dimension elements (several by lines)..." );
```

Comment:
```suggestion
  trace.beginBlock ( "Testing reading special 1 dimension elements (polygon of 1 dimension point)..." );
```

Reply:
Update done

Reply:
you could just click in commit suggestion given above ;)

Reply:
@Seangongx I think you miss it